config/pipeline.yaml

```diff
@@ -926,6 +926,15 @@ jobs:
 
   kbuild-gcc-12-arm64: *kbuild-gcc-12-arm64-job
 
+  kbuild-gcc-12-arm64-chromebook-main:
```

Comment:
It would make sense to me to have this one a plain "-chromebook" and the other one "-chromeos" (since it's in "*-chromeos.yaml" configs).

We can merge it as it is and do the rename as a next step (here, in jobs- and scheduler- files).

Reply:
Yes, that sounds fine to me. I wanted to keep this PR contained to pipeline.yaml and so I preferred doing the name change here. But renaming the chromeos ones in a separate PR sounds good.